### ISSUE TITLE
[Companion] Soul-Trader Beacon not generating ethereal credits.

### ISSUE THREAD

**Issue by Linozo** [dalaranwow/dalaran-wow#4394]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: When the companion http://www.wowhead.com/item=38050/soul-trader-beacon is spawned and you kill a mob which yields honor or exp it doesn't generate an ethereal credit as it should. It's just following you doing nothing. Also it's possible to SELL items to this companion which shouldn't be possible.

**Expected behaviour**: If the companion is summoned and you kill a mob which yields honor or exp it should generate an ethreal credit. http://www.wowhead.com/item=38186/ethereal-credit
You CANNOT sell to this merchant.

**Steps to reproduce the problem**:

1. Kill a mob which yields exp or honor with your companion summoned. You will get no ethereal credit after the kill.
2. ?
3. ?

**Include proofs for this behaviour**
This video shows how it should work: http://www.wowhead.com/npc=27914/ethereal-soul-trader#videos:id=9317

Will add video from current beviour soon.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/1249
